src/pages/workspaces/workspace/analysis/modals/AnalysisModal.js

```diff
@@ -297,8 +297,8 @@ export const AnalysisModal = withDisplayName('AnalysisModal')(
             onClick: async () => {
               try {
                 const contents = Utils.cond(
-                  [currentToolObj?.isNotebook, () => notebookData[notebookKernel]],
-                  [isRStudio, () => '# Starter file'])
+                  [currentToolObj?.isNotebook, () => JSON.stringify(notebookData[notebookKernel])],
+                  [isRStudio, () => baseRmd])
```

Comment:
It looks like `create` is expecting a [Blob](https://developer.mozilla.org/en-US/docs/Web/API/Blob) since it accesses `contents.size`, but here a string is being passed. This results in passing an undefined `Content-Length` header to `fetch`.